README.md

```diff
@@ -101,6 +93,64 @@ file.
 
 Note that new entry points will need the project to be unmounted and mounted again to be applied.
 
+### On the target machine (Wazo)
+
+#### Install dependencies
+Install rsync, and create a folder where repos must be available
+
+```sh
+apt update
+apt install rsync
+mkdir /usr/src/wazo  # or whatever your <local_source>
+```
+
+Install ssh
+```sh
+apt update
+apt install openssh-server
+```
+
+confiure ssh to enable root login
+```sh
+nano /etc/ssh/sshd_config
+```
+
+Add this line
+```sh
+PermitRootLogin yes
+```
+
+Restart sshd
+```sh
+sudo systemctl restart ssh
+```
```

Comment:
I think we assume ssh is configured, I feel adding this kind of instructions is out of scope of wdk and a prerequirement of wazo and linux server administration in general. We have to assume some things when documenting.